packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '@talend/react-components/lib/Icon';
+import classNames from 'classnames';
+import Widget from '../../Widget';
+
+import theme from './CollapsibleFieldset.scss';
+
+function defaultTitle(_, schema) {
+	return schema.title;
+}
+
+export default function createCollapsibleFieldset(title = defaultTitle) {
+	class CollapsibleFieldset extends React.Component {
+		constructor(props) {
+			super(props);
+			this.toggle = this.toggle.bind(this);
+		}
+
+		toggle(event) {
+			event.stopPropagation();
+			event.preventDefault();
+			const payload = {
+				schema: this.props.schema,
+				value: {
+					...this.props.value,
+					isClosed: !this.props.value.isClosed,
+				},
+			};
+			this.props.onChange(event, payload);
```

Comment:
the onChange props is required but has no default value
is it injected by the Form ?

Reply:
Yes it is injected by the forms. It's the callback that will change/consolidate the model